### PR TITLE
add button to show JSON

### PR DESCRIPTION
**Pull request by andrew-nowak**
## What does this change?

image

### CODE REVIEW

File: newswires/client/src/Item.tsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ export const Item = ({ id }: { id: string }) => {
 
 	const [itemData, setItemData] = useState<WireData | undefined>(undefined);
 	const [error, setError] = useState<string | undefined>(undefined);
+	const [showJson, setShowJson] = useState<boolean>(false);
```
Comment (twrichards) [inline newswires/client/src/Item.tsx:22]: pedantic, and you might disagree, but I find `should` or `is` prefixes useful for boolean variables (when I see `showJson` I assume that's a function...
```suggestion
	const [shouldShowJson, setShouldShowJson] = useState<boolean>(false);
```

File: newswires/client/src/Item.tsx [hunk 2]
```diff
@@ -79,10 +81,18 @@ export const Item = ({ id }: { id: string }) => {
...
-					<EuiFlyoutFooter>
+					<EuiFlyoutFooter
+						css={css`
+							display: flex;
+							justify-content: space-between;
+						`}
+					>
 						<EuiButton onClick={() => handleDeselectItem()}>Close</EuiButton>
+						<EuiButton onClick={() => setShowJson(!showJson)}>
+							Show JSON
```
Comment (twrichards) [inline newswires/client/src/Item.tsx:94]: nice to have...
```suggestion
							{shouldShowJson ? "Hide" : "Show"} JSON
```

Comment (twrichards) [reply newswires/client/src/Item.tsx:94]: unless EUI has a toggle button, in which case button text could be static

**Review by twrichards — APPROVED** [review]
LOVE this!

File: newswires/client/src/Item.tsx (hunk 2 above)
Comment (andrew-nowak) [reply newswires/client/src/Item.tsx:94]: ooh yeah toggle is a good shout! 
image

**Pull request merged**